R/predict.R

```diff
@@ -15,6 +15,15 @@
 #' @param min_date,max_date Optional. Date-like objects specifying the start
 #'   and end of the prediction range. See **Details** for more information on
 #'   their usage.
+#' @param day_of_week How to handle day-of-week effects when predicting
```

Comment:
slight drift in the docs between here and the first discussion - unless this doesn't try and use `reference_date` and instead tries to do something cleverer?

Reply:
It does try to do something cleverer. It looks to see if the day of week effect was used in the model fit. If yes it re-uses that effect. Then it checks to see if it can use the reference date. This pattern allows us to use custom day-of-week levels in the fit (e.g., "Holiday") and re-use them in the predict without requesting clarification from the user unless we need to ask (i.e., custom levels and a new date).